src/renderers/dom/client/wrappers/__tests__/ReactDOMTextarea-test.js

```diff
@@ -24,46 +25,51 @@ describe('ReactDOMTextarea', function() {
   beforeEach(function() {
     React = require('React');
     ReactDOM = require('ReactDOM');
+    ReactDOMServer = require('ReactDOMServer');
     ReactLink = require('ReactLink');
     ReactTestUtils = require('ReactTestUtils');
 
-    renderTextarea = function(component, container) {
+    renderTextarea = function(component, container, skipReplace) {
       if (!container) {
         container = document.createElement('div');
       }
-      var stub = ReactDOM.render(component, container);
-      var node = ReactDOM.findDOMNode(stub);
-      // Fixing jsdom's quirky behavior -- in reality, the parser should strip
-      // off the leading newline but we need to do it by hand here.
-      node.value = node.innerHTML.replace(/^\n/, '');
-      return stub;
+      var node = ReactDOM.render(component, container);
+
+      if (!skipReplace) {
```

Comment:
We should do this unconditionally but set node.defaultValue not node.value. Then the tests pass.